README.md

```diff
@@ -3,48 +3,17 @@
 A server that provides an REST API to interact with Postgres backups.
 
 ## Installation
-
-This tool is supposed to be installed in the same server where Barman is already
-set up and running.
-
-### Package installation
-
-Configure EDB repos 2.0 and install the package `pg-backup-api` using your
-package manager.
-
-### Installation from source code
-
-1. Clone this repository, e.g.:
-
-```bash
-git clone https://github.com/EnterpriseDB/pg-backup-api.git
-```
-
-2. Install `pg-backup-api` and its requirements:
-
-```bash
-cd pg-backup-api/pg_backup_api
-sudo python3 setup.py install
-```
-
-**Note:** `pg_config` application should be in your `PATH` because `psycopg2` is
-in the dependency tree. Besides that, you also need the `libpq` C library in
-your system (`libpq-dev` package on Debian based, or `libpq-devel` package on
-RedHat based).
+tbd
```

Comment:
Did you mean to revert the `README.md` changes here?

Reply:
No, something went wrong with the rebase, sorry.
I'm sure I accept head although it created problems with `README.md` 🤔 
I'll turn this into a draft and check what is going on.